frontend/apps/reader/modules/readersearch.lua

```diff
@@ -84,8 +83,10 @@ function ReaderSearch:onShowFulltextSearchInput()
                         self.last_search_text = self.input_dialog:getInputText()
                         self.use_regex = self.check_button_regex.checked
                         self.case_insensitive = not self.check_button_case.checked
-                        if self.use_regex and self.ui.document:checkRegex(self.input_dialog:getInputText()) ~= 0 then
-                            UIManager:show(InfoMessage:new{ text = _("Invalid regular expression") })
+                        local regex_error = self.ui.document:checkRegex(self.input_dialog:getInputText())
+                        if self.use_regex and regex_error ~= 0 then
+                            UIManager:show(InfoMessage:new{ text = _("Invalid regular expression.") })
+                            logger.dbg("ReaderSearch: regex error", tostring(regex_error))
```

Comment:
(You might be already working on this, but just mentionning it :)
I think you can have somewhere:
```lua
local SRELL_ERROR_CODES = {
  "100" = _("Collate something"),
  "105" = _("Missing parens"),
}
```
Good luck trying to understand what each of the 16 codes means :) Hope there exists somewhere in the code a good list with small explanations :)

Reply:
>Well, this is not the primary feature of the app, so I don't mind if we don't tell much (and go get all these warning messages translated !)

Did you change your mind?

Reply:
I don't mind if we don't tell much - but as you've done the work to forward the error code, I now mind that we don't use this valuable info :)
(If you find some short english strings in the official srell doc/source, I guess we can let them untranslated and displayed in english - might be better than bad translations.)